Core/Importers/KredoTransactionsImporter.cs

```diff
@@ -0,0 +1,64 @@
+﻿using iText.IO.Font;
+using iText.Kernel;
+using iText.Kernel.Font;
+using iText.Kernel.Pdf;
+using iText.Kernel.Pdf.Canvas.Parser;
+using iText.Kernel.Pdf.Canvas.Parser.Listener;
+using iText.Layout;
+using iText.IO.Font.Constants;
+using iText.Layout.Element;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+
+namespace Core.Importers
+{
+    class KredoTransactionsImporter : ITransactionsImporter
+    {
+        public IList<Transaction> Load(Stream file)
+        {
+            PdfDocument pdfDoc = new PdfDocument(new PdfReader(file));
+
+            FilteredEventListener listener = new FilteredEventListener();
+
+            ITextExtractionStrategy extractionStrategy = listener
+                .AttachEventListener(new SimpleTextExtractionStrategy());
+
+            var transactions = new List<Transaction>();
+            var pagesAmount = pdfDoc.GetNumberOfPages();
+            for (int i = 0; i < pagesAmount; i++)
+            {
+                new PdfCanvasProcessor(listener).ProcessPageContent(pdfDoc.GetPage(i + 1));
+
+                var actualText = extractionStrategy.GetResultantText().Split("\n").Skip(29).SkipLast(2).ToArray();
+
+                while (actualText.Length != 0)
+                {
+                    var transactionData = actualText.Skip(2).TakeWhile(s => Date.TryParse(s, out Date d) == false).ToArray();
```

Comment:
Codacy found an issue: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5961168)